input/samples/vn_simona/fullGrid/wec_type_input.csv

```diff
@@ -1,2 +1,2 @@
-"uuid","capex","cos_phi_rated","cp_characteristic","eta_conv","hub_height","id","opex","rotor_area","s_rated"
-b70374f3-1feb-45af-8011-1304a532d398,0.0,0.9,"cP:{(0.0,0.0),(1.0,0.0),(2.0,0.115933516),(3.0,0.286255595),(4.0,0.39610618),(5.0,0.430345211),(6.0,0.45944023),(7.0,0.479507331),(8.0,0.492113623),(9.0,0.500417188),(10.0,0.488466547),(11.0,0.420415054),(12.0,0.354241299),(13.0,0.288470591),(14.0,0.230965702),(15.0,0.18778367),(16.0,0.154728976),(17.0,0.128998552),(18.0,0.108671106),(19.0,0.09239975),(20.0,0.079221236),(21.0,0.068434282),(22.0,0.059520087),(23.0,0.052089249),(24.0,0.045845623),(25.0,0.040561273),(26.0,0.036058824),(27.0,0.032198846),(28.0,0.016618264),(29.0,0.010330976),(30.0,0.006091519),(31.0,0.003331177),(32.0,0.001641637),(33.0,0.000705423),(34.0,0.000196644),(35.0,0.0),(36.0,0.0),(37.0,0.0),(38.0,0.0),(39.0,0.0),(40.0,0.0),(41.0,0.0),(42.0,0.0),(43.0,0.0),(44.0,0.0),(45.0,0.0),(46.0,0.0),(47.0,0.0),(48.0,0.0),(49.0,0.0),(50.0,0.0)}",100.0,98.0,EnerconE-82,0.0,5281.02,2222.2
+uuid,capex,cos_phi_rated,cp_characteristic,eta_conv,hub_height,id,opex,rotor_area,s_rated
+b70374f3-1feb-45af-8011-1304a532d398,0.0,0.9,"cP:{(0.00,0.00),(1.00,0.00),(2.00,0.12),(3.00,0.29),(4.00,0.40),(5.00,0.43),(6.00,0.46),(7.00,0.48),(8.00,0.49),(9.00,0.50),(10.00,0.49),(11.00,0.42),(12.00,0.35),(13.00,0.29),(14.00,0.23),(15.00,0.19),(16.00,0.15),(17.00,0.13),(18.00,0.11),(19.00,0.09),(20.00,0.08),(21.00,0.07),(22.00,0.06),(23.00,0.05),(24.00,0.05),(25.00,0.04),(26.00,0.04),(27.00,0.03),(28.00,0.02),(29.00,0.01),(30.00,0.01),(31.00,0.00),(32.00,0.00),(33.00,0.00),(34.00,0.00),(35.00,0.00),(36.00,0.00),(37.00,0.00),(38.00,0.00),(39.00,0.00),(40.00,0.00),(41.00,0.00),(42.00,0.00),(43.00,0.00),(44.00,0.00),(45.00,0.00),(46.00,0.00),(47.00,0.00),(48.00,0.00),(49.00,0.00),(50.00,0.00)}",100.0,98.0,EnerconE-82,0.0,5281.02,2222.2
```

Comment:
It seems like here was some considerable rounding done .. find out why that is

Reply:
Depending on https://github.com/ie3-institute/PowerSystemDataModel/issues/710